Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -1444,15 +1444,15 @@ respondQuote (driverId, _) req = do
             createdAt = now,
             updatedAt = now,
             rideDetails =
-              DRide.RideDetailsRental
-                { rentalToLocation = Nothing,
-                  odometerStartReading = Nothing,
-                  odometerEndReading = Nothing,
-                  odometerStartReadingImagePath = Nothing,
-                  odometerEndReadingImagePath = Nothing,
-                  endRideOtp = Just endRideOtp
-                },
-            rideType = DRide.RENTAL
+              DRide.DetailsRental
+                DRide.RideDetailsRental
+                  { rentalToLocation = Nothing,
+                    odometerStartReading = Nothing,
+                    odometerEndReading = Nothing,
+                    odometerStartReadingImagePath = Nothing,
```

Comment:
@max-keviv Please make sure these paths are removed, we need to map it to `mediaFile` table